packages/web-components/src/divider/divider.ts

```diff
@@ -0,0 +1,38 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTDivider } from '@microsoft/fast-foundation';
+import { AlignContent, Appearance } from './divider.options.js';
+
+/**
+ * @class Divider component
+ *
+ * @remarks
+ * This class extends the FASTDivider. A divider groups sections of content to create visual rhythm and hierarchy. Use dividers along with spacing and headers to organize content in your layout.
+ */
+export class Divider extends FASTDivider {
+  /**
+   * @property alignContent
+   * @default center (AlignContent.center)
+   * @remarks
+   * Determines the alignment of the content within the divider. Select from center, start, or end.
+   */
+  @attr
+  public alignContent?: AlignContent = AlignContent.center;
```

Comment:
This should use the proper attribute syntax to hyphenate for attribute names as in html. We'll also want to remove the defaults and just set them on the host as "default". This will keep the DOM clean by not reflecting the value if it's not passed explicitly.

```ts
  @attr({ attribute: "align-content" })
  public alignContent?: AlignContent;
```